src/clr-addons/quick-list/quick-list.component.html

```diff
@@ -0,0 +1,22 @@
+<div [ngClass]="onCard?'card':''">
```

Comment:
the card styling should not be provided by this component, it should only be used in other contexts like a card